SR2025/2024-09-02-competition-announcement.md

```diff
@@ -0,0 +1,36 @@
+---
+to: Potential team supervisors
+subject: Announcing Student Robotics 2025!
+---
+
+Hi,
+
+We're pleased to [announce][announcement] that registration for Student Robotics
+2025 is now open. If you would like to compete, please complete the
+[signup form][signup-form] with the details of your team. All future emails
+about SR2025 will be sent to the details you provide there.
+
+This year we're hosting [Kickstart][kickstart] at the University of Southampton,
+on Saturday 12th October. We will also be live-streaming the event for those
+unable to attend in person. We expect to confirm places towards the latter half
+of September. Places are limited, so sign up soon to avoid disappointment.
+
+As usual we will loan competing teams a [kit][kit] of parts to help build their
+robots. This year we hope to send out kits before Kickstart.
+
+We have also prepared some [pre-Kickstart activities][pre-kickstart-activities]
+to enable teams to start learning about robotics and our simulator right away.
+
+  [I'm interested in bringing a team to SR2025][signup-form]
```

Comment:
I think breaking it into sections makes it easier to convey all the points we are trying to get across here.

```suggestion
We're pleased to [announce][announcement] that registration for Student Robotics
2025 is now open.

## Signups
If you would like to compete, please complete the
[signup form][signup-form] with the details of your team. All future emails
about SR2025 will be sent to the details you provide there.

  [I'm interested in bringing a team to SR2025][signup-form]

## Kickstart

This year we're hosting [Kickstart][kickstart] at the University of Southampton,
on Saturday 12th October. We will also be live-streaming the event for those
unable to attend in person. We expect to confirm places towards the latter half
of September. Places are limited, so sign up soon to avoid disappointment.

As usual we will loan competing teams a [kit][kit] of parts to help build their
robots. This year we hope to send out kits before Kickstart.

## Pre-Kickstart Activities

The [pre-Kickstart activities][pre-kickstart-activities] are a new concept we are introducing with SR2025. They use our simulator for a set of activities intended to be completed between signups and Kickstart. 
These will teach some core concepts of robotics and introduce some simple sensors that teams could use on their physical robots.

```

Reply:
Thanks for the suggestion. I can see what you're going for, though in this case I'm not sure that the suggested change is ideal. Among other things it moves the CTA to the top, which greatly reduces the chance that people will read the rest of the email (having it at the bottom was deliberate to counter this).

 The email is also pretty short, so I expect people will be able to read it all without needing it breaking up in to sections. I'll revisit this once we've got the links to the other general information in place and see if that changes the balance.